frontend/src/components/LogIn/login.module.css

```diff
@@ -0,0 +1,152 @@
+body {
+    background-color: #F4EFFA;
+    padding-top: 10%;
+    padding-bottom: 10%;
+}
+
+h1 {
+    letter-spacing: 5px;
+    padding-bottom: 2%;
+}
+
+.login_group {
```

Comment:
For all css the selectors should be lower kebab cased.

For example this should be .login-group {...}